master/master.cfg

```diff
@@ -336,6 +337,10 @@ def get_targets(os, llvm):
                         ('test_generators', t),
                         ('test_apps', t)])
 
+    # If Bazel is installed on the builder, run test_bazel
```

Comment:
I think this code runs on the master, not the builder. You probably just want to just condition it on the os starting with 'linux-64'